crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/service/MeansAssessmentService.java

```diff
@@ -34,4 +31,54 @@
             throw new AssessmentCriteriaNotFoundException(String.format("No Assessment Criteria found for date %s",assessmentDate));
         }
     }
+
+    public ApiCreateMeansAssessmentResponse createInitialAssessment(ApiCreateMeansAssessmentRequest meansAssessment){
+        log.info("Create initial means assessment - Start");
+        var response = TemporaryCreateMeansAssessmentReponse.getCreateMeansAssessmentResponse();
+
+        return response;
+    }
+
+    private static class TemporaryCreateMeansAssessmentReponse{
```

Comment:
why do we need to do this? It would be better to load the json file or would suggest supporting this with a test?